### ISSUE TITLE
DOCX custom-style for lists and tables

### ISSUE THREAD

**Issue by bpj** [jgm/pandoc#4697]
There have been several questions on pandoc-discuss lately on styling lists and tables in DOCX. This is currently not possible: If you wrap a list or table in a div with `custom-style=Foo` this results in applying the *paragraph* style `Foo` to all the items in the list or cells in the table, which probably seldom is what the user expects. What would it take to apply *list* style `Foo` or *table* style `Foo` in these cases? I guess @jkr is the person who could answer this.

**Comment by agusmba** [comment]
@bpj see related issues: https://github.com/jgm/pandoc/issues/2667, https://github.com/jgm/pandoc/issues/2033, https://github.com/jgm/pandoc/issues/774

**Comment by mb21** [comment]
Looks like we should close this issue (and others?) in favour of a single one?

**Comment by agusmba** [comment]
I agree with not having multiple duplicate issues. Two of them have already been closed. I see two relevant ones open (maybe not as concise as this one, but with more discussion).

 #2667 deals with grid table formatting but indirectly also with table styles

#1024 has been mentioned elsewhere as a way of introducing necessary changes to support adding custom styles to tables (different from the official "Table") Note that this issue does not deal with custom-style for tables directly

**Comment by jsommer1738** [comment]
Is there any progress on this?

**Comment by tarleb** [comment]
> Is there any progress on this?

I don't know what kind of answer you expect. If this is a way of saying "go work on it", then please refrain from asking any such question in the future.

**Comment by jsommer1738** [comment]
It certainly wasn't meant as a way of passively aggressively asking someone to work on it.  Maybe a better way to ask would have been, "Is this ticket scheduled for a specific release?"

In general, I can see the ticket is open but I have no visibility into when it might get worked.  And, no, I am in no way trying to pressure anyone to do work.

**Comment by mb21** [comment]
it's all volunteers here :-) so there is no schedule. either somebody comes along and makes a pull request, or at some point one of the core devs finds a day to burn... but no telling when that day arrives :)

**Comment by jsommer1738** [comment]
Understood.  Thanks!
